src/env.ts

```diff
@@ -74,6 +74,10 @@ export const env = createEnv({
     // Analytics
     NEXT_PUBLIC_GTM_ID: z.string().min(1).optional(),
     NEXT_PUBLIC_AMPLITUDE_API_KEY: z.string().min(1).optional(),
+    // Inkeep
+    NEXT_PUBLIC_INKEEP_API_KEY: z.string().min(1).default('b0537306817fb8a0daea377df2b273d1b00ac709182d1dc7'),
```

Comment:
Is it ok that we have default value for secrets?

Reply:
@JuroUhlar any ideas about what to do with it?

Reply:
I think it's okay, it's also okay to remove them, whatever you prefer. These are public values, you can always get to them by _Inspect element_. In the browser. Storing them just in env variables makes it harder to find them for other people, also for us if we need to test this functionality locally because you need to get the values first.

Reply:
+1. let's keep as is then. We can have it merged later today.